docs/preview/features/writing-different-telemetry-types.md

```diff
@@ -140,10 +141,29 @@ var durationMeasurement = new Stopwatch();
 durationMeasurement.Start();
 var startTime = DateTimeOffset.UtcNow;
 
-_logger.logger.LogIotHubDependency(iotHubConnectionString: "Hostname=sensors;", isSuccessful: true, startTime: startTime, duration: durationMeasurement.Elapsed);
+logger.LogIotHubDependency(iotHubConnectionString: "Hostname=sensors;", isSuccessful: true, startTime: startTime, duration: durationMeasurement.Elapsed);
 // Output: "Dependency Azure IoT Hub named sensors in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: )"
 ```
 
+### Measuring Azure Key Vault dependencies
+
+We allow you to measure Azure Key vault dependencies.
+
+**Example**
+
+Here is how you can report a dependency call:
+
+```csharp
+var durationMeasurement = new StopWatch();
+
+// Start measuring
+durationMeasurement.Start();
+var startTime = DateTimeOffset.UtcNow;
+
+logger.AzureKeyVaultDependency(vaultUri: "https://my-secret-store.vault.azure.net", operationName: "get secret", isSuccessful: true, startTime: startTime, duration: durationMeasurement.Elapsed);
```

Comment:
This should also include the secret name that was consumed

Reply:
Besides the 'operation name' or instead of the operation name ?

Reply:
I removed the operation name and replaced it with the secret name as it will probably make more sense to pass along the name of the secret then then name of the operation (like 'getting secrets' or 'setting secret').